versions/arabica_versions.js

```diff
@@ -0,0 +1 @@
+const arabicaVersions = Object.freeze({ "golang": "1.19.1", "app-latest-tag": "v0.11.1", "app-latest-sha": "a1eb594d43137ded5f9c1062168d26139a251e16", "core-latest-tag": "v1.14.0-tm-v0.34.23", "core-latest-sha": "ab4ef71b202dd9d27c879a820fa06e127d4234fa", "node-latest-tag": "v0.7.0-rc1", "node-latest-sha": "6933f03feb4a35ecd018e1a3cd70d7ae1644eb51" }); export default arabicaVersions;
```

Comment:
as a follow up, we could install prettier in the github action and run it after writing these files and before submitting the PR.  Then this will be nicely formatted.

Reply:
going to implement this before we merge. having to click "load diff" on the PR code view is not ideal

Reply:
successfully tested on https://github.com/jcstein/docs/pull/20/files